src/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -0,0 +1,45 @@
+package dotty.communitybuild
+
+import java.nio.file.{Path, Paths}
+
+import org.junit.{Ignore, Test}
+
+class CommunityBuildTest extends BuildTest {
+  def testDirectory: Path =
+    Paths.get(sys.props("user.dir") + "/out/")
+
+  @Test def algebra = project(
+    gitUrl = "https://github.com/dotty-staging/algebra",
+    command = "coreJVM/compile"
+  )
+
+  @Test def `better-files` = project(
```

Comment:
It's a bit annoying to do the `$minus`, maybe it's better to use `betterFiles`.

Reply:
Or, just `betterfiles` and use the same convention for all projects, lowercase + no separator (or `_` as separator).